tst/EnergyPlus/unit/ICSCollector.unit.cc

```diff
@@ -19,8 +20,9 @@ using namespace EnergyPlus::ConvectionCoefficients;
 using namespace EnergyPlus::DataSurfaces;
 using namespace EnergyPlus::DataHeatBalance;
 using namespace EnergyPlus::DataHeatBalSurface;
+using namespace EnergyPlus::Psychrometrics;
 
-TEST( ISCSolarCollectorTest, BugTest ) {
+TEST( ISCSolarCollectorTest, CalcPassiveExteriorBaffleGapTest ) {
 
```

Comment:
@jasondegraw This includes a name change for the ICS unit test, per your request.